django_netjsonconfig/static/django-netjsonconfig/css/admin.css

```diff
@@ -96,3 +96,7 @@
 #container .help a,
 #netjsonconfig-hint a{ text-decoration: underline }
 #netjsonconfig-hint{ width: auto }
+
+p.help{
```

Comment:
We just wanted the text "Choose items and order by drag & drop" to be bold, not every help text.

Also, making it bolder would probably not be enough to make it more evident, we should make it darker.